streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -939,6 +1209,7 @@ private void addStreamTasks(final Collection<TopicPartition> assignment, final l
         final Map<TaskId, Set<TopicPartition>> newTasks = new HashMap<>();
 
         // collect newly assigned tasks and reopen re-assigned tasks
+        log.info("{} Resuming suspended re-assigned tasks", logPrefix);
```

Comment:
This seems a bit misleading, and less informative. Maybe just "Adding active tasks {partitionAssignor.activeTasks()}"?

Ditto below for standby tasks